scheduler/docs/configuration.adoc

```diff
@@ -375,6 +376,11 @@ The configuration values are defined as follows:
   An integer representing the number of millisecond intervals at which progress updates will be published to datomic.
   The default value is 2500.
 
+`:sequence-cache-threshold`::
+  An integer representing the size of the task id to progress sequence cache to maintain.
+  This cache is used to track the latest sequence number of progress message processed for a given task.
+  The default value is 1000.
+
```

Comment:
In order to avoid the potential for out-of-order progress updates, this cache should be sized to handle the maximum number of active tasks that are reporting progress.

Reply:
Done.